src/domain/event/CreateEventPage.tsx

```diff
@@ -119,7 +122,14 @@ const CreateEventPage: React.FC = () => {
       await clearApolloCache();
       history.push({
         pathname: `/${locale}${ROUTES.CREATE_OCCURRENCE.replace(':id', id)}`,
+        search: firstOccurrencePrefilledValuesToQuery(values),
       });
+
+      // TODO: think if there is better way to handle this scrollTop.
+      // at the moment <ScrollToTop /> component ignores
+      // '/events/:id/occurrences/create' because it matches one of the ignored paths:
+      // '/events/:id/occurrences/:occurrenceId'
+      window.scrollTo(0, 0);
```

Comment:
Could you add new prop to ScrollToTop where you define array of routes that should alway scroll to top. In that case you could /events/:id/occurrences/:occurrenceId to ignored routes and /events/:id/occurrences/create to the routes (I don't know what could be suitable name for that prop) that should always scroll to top

Reply:
added new prop to ScrollToTop component and use that to handle the scrolling